# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/bc/use-toolchain-bc.patch.cross
# Copyright (C) 2022 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- bc-1.08.1/bc/Makefile.am.vanilla	2025-01-30 18:18:35.054994005 +0100
+++ bc-1.08.1/bc/Makefile.am	2025-01-30 18:19:06.752992741 +0100
@@ -37,8 +37,7 @@
 	$(LINK) -o $@ $(fbcOBJ) global_bootstrap.o $(LIBBC) $(LIBL) $(READLINELIB) $(LIBS)
 
 libmath.h: libmath.b
-	$(MAKE) fbc-bootstrap
-	./fbc-bootstrap -c $(srcdir)/libmath.b </dev/null | \
+	bc -c $(srcdir)/libmath.b </dev/null | \
 		sed -f fix-libmath.sed \
 		> libmath.tmp
 	mv libmath.tmp libmath.h
